clients/privacy-center/__tests__/server-utils/safelyGetUrl.test.ts

```diff
@@ -0,0 +1,25 @@
+import { safelyGetHost } from "~/app/server-utils/safelyGetHost";
```

Comment:
Sorry, I missed this earlier. Since the filename was changed, this test file should also probably be renamed.